examples/README.md

```diff
@@ -81,6 +106,25 @@ npx tsc examples/01-basic-usage.ts --outDir dist
 node dist/01-basic-usage.js
 ```
 
+### JSON Schema Integration Usage
+
+The JSON Schema integration provides a powerful way to automatically generate factories:
+
+```typescript
+import { createFactoryFromJsonSchema } from 'interface-forge/json-schema';
```

Comment:
So, this should be a factory class, not a helper. 

I'll comment in the code directly

Reply:
please address this comment